docs/appendix/zowe-glossary.md

```diff
@@ -0,0 +1,188 @@
+
+  The Zowe Application Framework modernizes and simplifies working on the mainframe. With the Zowe Application Framework, you can create applications to suit your specific needs. It is a web user interface (UI) that provides a virtual desktop containing a number of apps allowing access to z/OS function. 
+
+* **Zowe CLI**
+* **Zowe Mobile**
+* **Zowe Explorer** 
+* **Zowe Client SDKs** 
+* **Zowe Embedded Browser for RMF/SMF and APIs (ZEBRA)** 
+
+  Provides re-usable and industry compliant JSON formatted RMF/SMF data records, so that many other ISV SW and users can exploit them using open-source SW for many ways. For more information, see the ZEBRA documentation or visit the ZEBRA test/trial site. 
+       
+* **Zowe Workflow WiZard**
+
+* **Zowe Launcher**
+
+## Architecture
+
+* **Imperative** 
+
+  It's the framework powering Zowe CLI and Zowe Explorer.
+
+* **Explorer** 
+
+  It's the plugin suite for popular client-side editors such as VSCode.
+
+* **Web Explorer**
+
+  It's a suite of websites which is part of the Zowe server installation. It includes the JES, MVS, USS, and IP Explorers. It's not related to the "Explorer" which can be confusing.
+
+* **ZLUX**  
+
+  This is a codename for the Zowe Application Framework, consisting of the Zowe desktop and App server. We should replace as many occurrences of "ZLUX" in our documentation with "desktop", "app framework" and "app-server" according to which part we are talking about. More mentions of ZLUX lead to more confusion. 
+
+* **API Gateway**  
+
+  The API Gateway is a proxy server that routes requests from clients on its northbound edge, such as web browsers or the Zowe command line interface, to servers on its southbound edge that are able to provide data to serve the request. The API Gateway is also responsible for generating the authentication token used to provide single sign-on (SSO) functionality. See https://docs.zowe.org/stable/getting-started/zowe-architecture/#api-gateway
+
+* **API Catalog** 
+
+  The API Catalog provides a list of the API services that have registered themselves as catalog tiles. 
+
+* **API Discovery** 
+
+  a service to allow RESTful services to be dynamically added to the API Catalog
+
+* **App Server** 
+
+  The App Server is a node.js server that is responsible for the Zowe Application Framework. This server provides the Zowe desktop, which is accessible through a web browser. The Zowe desktop includes a number of applications that run inside the Application Framework such as a 3270 emulator and a File Editor.     
+
+* **Caching Service** 
+
+  This feature is designed for Zowe components in a high availability configuration. It supports the High Availability of all components within Zowe, allowing components to be stateless by providing a mechanism to offload their state to a location accessible by all instances of the service, including those which just started 
+       
+* **Zowe Cross Memory Server (X-MEM)** - V1: 
```

Comment:
Like ZLUX, nothing should say X-MEM.
It is "ZIS" in our documentation, to prevent confusion.